Tests/test_imagecms.py

```diff
@@ -174,19 +174,24 @@ def test_exceptions():
     psRGB = ImageCms.createProfile("sRGB")
     pLab = ImageCms.createProfile("LAB")
     t = ImageCms.buildTransform(pLab, psRGB, "LAB", "RGB")
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="mode mismatch"):
         t.apply_in_place(hopper("RGBA"))
 
     # the procedural pyCMS API uses PyCMSError for all sorts of errors
     with hopper() as im:
-        with pytest.raises(ImageCms.PyCMSError):
+        with pytest.raises(ImageCms.PyCMSError, match="cannot open profile file"):
             ImageCms.profileToProfile(im, "foo", "bar")
-    with pytest.raises(ImageCms.PyCMSError):
+
+    with pytest.raises(ImageCms.PyCMSError, match="cannot open profile file"):
         ImageCms.buildTransform("foo", "bar", "RGB", "RGB")
-    with pytest.raises(ImageCms.PyCMSError):
+
+    with pytest.raises(ImageCms.PyCMSError, match="Invalid type for Profile"):
         ImageCms.getProfileName(None)
     skip_missing()
-    with pytest.raises(ImageCms.PyCMSError):
+
+    # macOS/Ubuntu: "'NoneType' object cannot be interpreted as an integer"
+    # Windows: "an integer is required (got type NoneType)"
```

Comment:
This doesn't seem [correct.](https://github.com/radarhere/Pillow/runs/6953537293?check_suite_focus=true#step:24:1570)

Reply:
Ah, interesting, different messages on Windows depending on version.

3.7-3.9:

```
E           PIL.ImageCms.PyCMSError: an integer is required (got type NoneType)
```

https://github.com/radarhere/Pillow/runs/6953537155?check_suite_focus=true#step:24:1570
https://github.com/radarhere/Pillow/runs/6953537211?check_suite_focus=true#step:24:1570
https://github.com/radarhere/Pillow/runs/6953537251?check_suite_focus=true#step:24:1570

3.10:

```
E           PIL.ImageCms.PyCMSError: 'NoneType' object cannot be interpreted as an integer
```

https://github.com/radarhere/Pillow/runs/6953537293?check_suite_focus=true#step:24:1570